README.md

```diff
@@ -145,7 +145,29 @@ spec:
     image: mysql
 ```
 
-#### Cloud Servers ( **WIP** )
+#### Cloud Servers
+```Note: RDS DB instance will be provisioned manually by deployment team and login
```

Comment:
nit: use ** instead of ```
**Note: RDS DB instance will be provisioned manually by deployment team and login credentials will be provided to user.**

Reply:
rephrase:
**NOTE: RDS database server needs to be provisioned manually and user should get login credentials.**